src/dash-table/components/Table/props.ts

```diff
@@ -193,6 +198,7 @@ export interface IBaseColumn {
     clearable?: boolean | boolean[] | 'first' | 'last';
     deletable?: boolean | boolean[] | 'first' | 'last';
     editable: boolean;
+    filter_options: FilterCase;
```

Comment:
FilterCase vs. `filter_options` may look inconsistent but is intended to be that way. In the future you may also want to add locale-aware comparisons and leaving some space to do so at a future time.